src/js/structs/Transaction.js

```diff
@@ -1,5 +1,12 @@
+import FormActionTypes from '../constants/TransactionTypes';
```

Comment:
Maybe we should make this variable `TransactionTypes` just to keep things consistent.